onnxscript/converter.py

```diff
@@ -1321,7 +1310,17 @@ def translate_block(self, stmts, name, live_defs, parent_stmt=None):
                     self.current_fn, ovar, typeinfo, debuginfo.DebugInfo(info_stmt, self)
                 )
         graph = self.exit_scope()
-        return graph.to_graph_proto()
+        return graph.to_graph_and_functions()
+
+    def translate_nested_function_def(self, fn: ast.FunctionDef):
+        """
+        Translate a nested function definition.
+        """
```

Comment:
```suggestion
        """Translate a nested function definition."""
```

nit: style preference. https://google.github.io/styleguide/pyguide.html#383-functions-and-methods